scripts/startup/start_peas.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+WINDOW="${1}"
+echo "Running $(basename "${0}") at $(date), WINDOW=${WINDOW}"
+
+# Wait for bash to be ready (not necessary, but makes
+# the window look tidier when you attach later).
+sleep 1s
+
+tmux send-keys -t "${WINDOW}" "date" C-m
+tmux send-keys -t "${WINDOW}" "cd $POCS" C-m
+tmux send-keys -t "${WINDOW}" "bin/peas_shell" C-m
+sleep 10s
+tmux send-keys -t "${WINDOW}" "display_config" C-m
+sleep 1s
+tmux send-keys -t "${WINDOW}" "load_environment" C-m
+sleep 5s
+tmux send-keys -t "${WINDOW}" "load_weather" C-m
+sleep 5s
+tmux send-keys -t "${WINDOW}" "start" C-m
+sleep 20s
+tmux send-keys -t "${WINDOW}" "status" C-m
+sleep 10s
+tmux send-keys -t "${WINDOW}" "last_reading environment" C-m
+sleep 10s
+tmux send-keys -t "${WINDOW}" "last_reading weather" C-m
```

Comment:
Note that the first weather reading usually pulls across `Undefined` for all the values which I suppose could cause alarm when shown here.

Reply:
How often do we get readings? 30 seconds? A minute? As it stands, it runs about 40 seconds after starting.

Reply:
The first reading takes longer and comes back with the undefined values. I find it's usually about 2 minutes before any valid weather reading. Could just leave as is and we can watch it to see if it looks okay.

Reply:
Sure, will leave as is.